src/flows/context/variables.tsx

```diff
@@ -37,8 +36,7 @@ export const VariablesProvider: FC = ({children}) => {
       getRangeVariable(TIME_RANGE_START, range),
       getRangeVariable(TIME_RANGE_STOP, range),
     ]
-    const windowVar = getWindowPeriodVariableFromVariables(source, timeVars)
-    setVariables(!!windowVar ? timeVars.concat(windowVar) : timeVars)
```

Comment:
windowVars (for the editor) are now handled in the prelude to the LSP.
Benefiting all editors across the app.